packages/wp-now/src/tests/wp-now.spec.ts

```diff
@@ -859,4 +859,4 @@ describe('wp-cli command', () => {
 		await executeWPCli(['cli', 'version']);
 		expect(output).toMatch(/WP-CLI (\d\.?)+/i);
 	});
-});
+}, 30_000);
```

Comment:
Why increase this value?

Reply:
I was getting test-timeouts with the default.